### PR TITLE
Create nepali_phonetic_npl.xml

### PR DESCRIPTION
**Pull request by Shareef101**
nepali phonetic keyboard

### CODE REVIEW

File: res/xml/nepali_phonetic_npl.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="Nepali phonetic" script="nepali">
```
Comment (Julow) [inline res/xml/nepali_phonetic_npl.xml:2]: The Napali language uses the Devanagari script ? This should be `script="devanagari"`, the filename should be `deva_phonetic_npl.xml`

File: res/xml/nepali_phonetic_npl.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  <row>
+    <key key0="ए" key2="ऐ" key4="esc" key7="१"/>
+    <key key0="उ" key2="ऊ" key3="\@" key7="२"/>
```
Comment (Julow) [inline res/xml/nepali_phonetic_npl.xml:5]: Layout includes some ASCII punctuation but not all, missing: `.`, `<`, `>`, `?`, `` ` ``, `|`, `~`.
Are these useful in your language ?

**Review by Julow — COMMENTED** [review]
You need to run `python gen_layouts.py` to update the generated files.

File: res/xml/nepali_phonetic_npl.xml (hunk 2 above)
Comment (Shareef101) [reply res/xml/nepali_phonetic_npl.xml]: i added some of the punctuation you suggested and full stop "." Is not used in nepali.

**Comment by Shareef101** [comment]
i don't know how to run the python code that you told me. Help!

File: res/xml/nepali_phonetic_npl.xml (hunk 2 above)
Comment (Julow) [reply res/xml/nepali_phonetic_npl.xml]: Add only what is useful!

File: res/xml/nepali_phonetic_npl.xml (hunk 1 above)
Comment (Julow) [reply res/xml/nepali_phonetic_npl.xml]: I meant to change the `script` field, not the `name` field.

**Comment by Julow** [comment]
I would like to merge this soon. Is this ready ?

**Comment by Julow** [comment]
Any updates ?

**Comment by Julow** [comment]
Any news ? I would not merge a layout that have no user.

**Comment by Julow** [comment]
I'll close this PR as it seems abandoned. Feel free to re-open if that's not the case.

**Pull request closed without merging**